### PR TITLE
Bb 17 navbar component

### PR DESCRIPTION
**Pull request by brunobmv**
## Describe your changes:
Created navbar component that can be used in every page (just need to add it to the JSX)

## Proof that your changes work (include screenshots and/or for frontend):
Screen Shot 2023-04-05 at 17 01 15

## Jira Issue ticket number and link:
https://bearbuddies.atlassian.net/browse/BB-17?atlOrigin=eyJpIjoiZDNjNDgxM2NhZTdmNGY3NmE3YTcwZmQzNjJhNDkyMWEiLCJwIjoiaiJ9
BB-17

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have DM'd the group chat that I have made a PR and need reviewers.
- [x] I haved moved the Jira issue to "Under Review"
- [x] I have run `npx eslint filename.js` on all JavaScript files I've changed in this PR and fixed all displayed errors
- [x] I have checked the Chrome console in Developer Tools to make sure there are no significant errors

### CODE REVIEW

File: src/components/NavBar/NavBar.module.css
Comment (DylanHopkins21) [inline src/components/NavBar/NavBar.module.css:1]: idk if we have specific styling for the css files (if it should be alphabetical or smth) if not then this is chill

File: src/pages/SignupPage/SignupPage.js
Comment (DylanHopkins21) [inline src/pages/SignupPage/SignupPage.js:1]: trusting this is all good, no idea what you actually changed here. Next time don't include unrelated files in your PR

**Review by DylanHopkins21 — CHANGES_REQUESTED** [review]
Is there any chance you could make the text blue onHover? just would feel much better

File: src/pages/SignupPage/SignupPage.js
Comment (thomaspwang) [reply src/pages/SignupPage/SignupPage.js:1]: @brunobmv what did you need to change in signup? generally you want to describe any changes in the PR

**Review by thomaspwang — APPROVED** [review]
looks good to me! just change what dylan requested (blue hover)

**Comment by brunobmv** [comment]
Fixed all suggestions. no more changes to signup which i just used to visualize my changes to the component, and the hover works well. I think it is ready to merge to main

**Review by DylanHopkins21 — APPROVED** [review]
Looks fire, good job!

**Pull request merged**